addons/l10n_pt/data/account_chart_template_data.xml

```diff
@@ -11,10 +11,12 @@
         <field name="property_account_income_categ_id" ref="chart_711"/>
         <field name="property_account_expense_categ_id" ref="chart_311"/>
         <field name="income_currency_exchange_account_id" ref="chart_7861"/>
-        <field name="expense_currency_exchange_account_id" ref="chart_692"/>
+        <field name="expense_currency_exchange_account_id" ref="chart_6928"/>
```

Comment:
`chart_6928,6928,503,Others,expense,l10n_pt.pt_chart_template,False`
Others? It looks incorrect